rosdep/python.yaml

```diff
@@ -2560,6 +2560,10 @@ python-telegram-bot:
   ubuntu:
     pip:
       packages: [python-telegram-bot]
+python-tensorflow:
```

Comment:
Please use `-pip` extension for pip only packages. Pip is often well ahead of the system packages when they eventually appear in the future which can cause future breakage.